qt/aqt/tts.py

```diff
@@ -606,13 +604,10 @@ def _on_done(self, ret: Future, cb: OnDoneCallback) -> None:
             if exception := ret.exception():
                 print(str(exception))
                 tooltip(tr.errors_windows_tts_runtime_error())
-                cb()
```

Comment:
It's not safe to remove this - if you inject an error into _play() above, you'll see that if cb is not called, no further sounds can play, as Anki never notices that the previous play request has failed/finished.